Activated option "Automatic Coercions Import" so that ReleasedSsreflect
compile after Coq v8.3 revision 14776 fixed unexpected import of
coercions in innermost modules of a chain of nested module when
only outermost module is imported.

--- ssreflect-1.3pl2-old/theories/ssreflect.v	2011-03-25 05:12:49.000000000 -0400
+++ ssreflect-1.3pl2/theories/ssreflect.v	2012-02-09 18:05:35.005771434 -0500
@@ -3,6 +3,7 @@
 Require Import Bool. (* For bool_scope delimiter 'bool'. *)
 
 Declare ML Module "ssreflect".
+Global Set Automatic Coercions Import.
 
 (***************************************************************************)
 (* This file is the Gallina part of the ssreflect plugin implementation.   *)
